### PR TITLE
EXE & DDL: Migrate project files to VS2015, drop Unity build targets.

### PR DESCRIPTION
**Pull request by Warepire**
Keep up with the compiler, and rip out the Unity build targets, because all they do is complicate the creation of new files for very little gain.

### CODE REVIEW

**Comment by jroweboy** [comment]
LGTM

File: wintasee/wintasee2015.vcxproj [hunk 1]
```diff
@@ -0,0 +1,219 @@
...
+    <ProjectGuid>{948D0C09-EE11-4241-9286-D88C5B84DA36}</ProjectGuid>
+    <RootNamespace>wintasee</RootNamespace>
+    <Keyword>Win32Proj</Keyword>
+    <WindowsTargetPlatformVersion>8.1</WindowsTargetPlatformVersion>
+  </PropertyGroup>
+  <Import Project="$(VCTargetsPath)\Microsoft.Cpp.Default.props" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
+    <ConfigurationType>DynamicLibrary</ConfigurationType>
+    <CharacterSet>MultiByte</CharacterSet>
+    <PlatformToolset>v140</PlatformToolset>
```
Comment (YaLTeR) [inline wintasee/wintasee2015.vcxproj]: Switch this to v140_xp as well perhaps?

File: wintaser/wintaser2015.vcxproj [hunk 2]
```diff
@@ -23,35 +15,22 @@
...
-    <CharacterSet>MultiByte</CharacterSet>
-  </PropertyGroup>
-  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='DebugUnity|Win32'" Label="Configuration">
-    <ConfigurationType>Application</ConfigurationType>
-    <CharacterSet>MultiByte</CharacterSet>
-  </PropertyGroup>
   <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Release|Win32'" Label="Configuration">
     <ConfigurationType>Application</ConfigurationType>
     <CharacterSet>MultiByte</CharacterSet>
+    <PlatformToolset>v140</PlatformToolset>
```
Comment (YaLTeR) [inline wintaser/wintaser2015.vcxproj]: And here v140 into v140_xp

**Comment by Warepire** [comment]
This also takes care of issue #12

**Comment by Warepire** [comment]
Merging this in the weekend unless you have any further objections @YaLTeR.

**Pull request merged**